examples/python/src/main/java/com/hazelcast/jet/examples/python/Python.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.examples.python;
+
+import com.hazelcast.internal.nio.IOUtil;
+import com.hazelcast.jet.Jet;
+import com.hazelcast.jet.JetInstance;
+import com.hazelcast.jet.Observable;
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.function.Observer;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.test.TestSources;
+import com.hazelcast.jet.python.PythonServiceConfig;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Objects;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Stream;
+
+import static com.hazelcast.jet.impl.util.Util.uncheckRun;
+import static com.hazelcast.jet.python.PythonTransforms.mapUsingPython;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * This example shows you how to invoke a Python function to process the
```

Comment:
I think we should say what the pipeline does, rather than just "process data". It's better to lead by describing the actual processing that is done